src/lib/puzzleEndpoints.js

```diff
@@ -0,0 +1,35 @@
+import { httpGet, httpPut } from "./dataAccess";
+
+const baseEndpoint = process.env.REACT_APP_ADMIN_BASE_ENDPOINT;
+
+export const getPuzzle = async (sceneId, objectId, handleError) => {
+    try {
+        const response = await httpGet(
+            baseEndpoint + `scene/${sceneId}/object/${objectId}/puzzle`
+        );
+        return response.data;
+    } catch (error) {
+        handleError(error);
+        throw error;
+    }
+};
+
+export const editPuzzle = async (
+    { sceneId, objectId, isInteractable, animationsJson },
+    handleError
+) => {
+    try {
+        const response = await httpPut(
+            baseEndpoint + `scene/${sceneId}/object/${objectId}/puzzle`,
+            {
+                is_interactable: isInteractable,
+                animations_json: animationsJson,
+            }
+        );
+        alert("Saved puzzle CRUD changes for object with id: " + objectId);
```

Comment:
This should not be done in the puzzleEndpoints. Should be done in the function that calls "editPuzzle"

Other flows might call this that aren't explicit (to the user) that they are editing a puzzle